src/Widgets/PasswordEditor.vala

```diff
@@ -32,27 +32,25 @@ namespace SwitchboardPlugUserAccounts.Widgets {
         private PasswordQuality.Settings pwquality;
 
         public bool is_authenticated { public get; private set; }
```

Comment:
If you use Object () on this, you should consider having the signature like this:
`public bool is_authenticated { get; construct set; }`

This will make the member accessible for setting from other classes thoguh, so you should be careful in changing that. I guess you should just set a `default = false`. If you go with this approach, you don't have to set anything in the constructor.